Source/CombatTrainingMod/JobDriver_TrainCombat.cs

```diff
@@ -182,22 +189,18 @@ private void TryGainCombatTrainingRoomThought()
         }
 
         /*
-         * Returns true if learning has been saturated for today
+         * Returns the current skill: shooting or melee.
          */
-        private bool LearningSaturated()
```

Comment:
Probably should have left this method and moved the logic for learning saturation in regards to the training in this method.  I'll update the code.